src/main/scala/com/newlibertie/pollster/impl/PollParameters.scala

```diff
@@ -23,4 +23,18 @@ case class PollParameters
   if(creation_ts == None) {
     creation_ts = Some(new Date())
   }
+  override def toString: String = {
```

Comment:
can we use a more specific method of a more specific name to do the transformation we are doing in toString

Reply:
Same comment as above: How about using toJsonString? Else can you suggest a name as it is used to return Poll object to the REST client?